3.x Types Issue: FontAwesomeIconProps missing values

**Describe the bug**
Types in [index.d.ts FontAwesomeIconProps](https://github.com/FortAwesome/vue-fontawesome/blob/3.x/index.d.ts#L15-L20) do not match actual expected props in [FontAwesomeIcon.js](https://github.com/FortAwesome/vue-fontawesome/blob/3.x/src/components/FontAwesomeIcon.js#L80-L136).

**Expected behavior**
Exact match of supported props and associated types definition.

**Reproduction**
In any typescript app consuming v3.0.2 of vue-fontawesome, attempt to utilize prop size="xl". See type error:
`error TS2322: Type '"xl"' is not assignable to type '"lg" | "xs" | "sm" | "1x" | "2x" | "3x" | "4x" | "5x" | "6x" | "7x" | "8x" | "9x" | "10x" | undefined'.`

**Additional context**
Issue introduced by: https://github.com/FortAwesome/vue-fontawesome/pull/366

**Exact diff of expected types**
```diff
- size?:         'xs' | 'sm' | 'lg'                | '1x' | '2x' | '3x' | '4x' | '5x' | '6x' | '7x' | '8x' | '9x' | '10x'
+ size?: '2xs' | 'xs' | 'sm' | 'lg' | 'xl' | '2xl' | '1x' | '2x' | '3x' | '4x' | '5x' | '6x' | '7x' | '8x' | '9x' | '10x'
+ bounce?: boolean
+ shake?: boolean
+ beat?: boolean
+ fade?: boolean
+ beatFade?: boolean
+ flash?: boolean
+ spinPulse?: boolean
+ spinReverse?: boolean
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -12,7 +12,7 @@ interface FontAwesomeIconProps {
   pulse?: boolean
   rotation?: 90 | 180 | 270 | '90' | '180' | '270'
   swapOpacity?: boolean
-  size?: 'lg' | 'xs' | 'sm' | '1x' | '2x' | '3x' | '4x' | '5x' | '6x' | '7x' | '8x' | '9x' | '10x'
+  size?: '2xs' | 'xs' | 'sm' | 'lg' | 'xl' | '2xl' | '1x' | '2x' | '3x' | '4x' | '5x' | '6x' | '7x' | '8x' | '9x' | '10x'
   spin?: boolean
   transform?: object | string
   symbol?: boolean | string

